snapshot/cmd/snapshot-controller/snapshot-controller.go

```diff
@@ -36,9 +36,9 @@ import (
 	"github.com/kubernetes-incubator/external-storage/snapshot/pkg/cloudprovider/providers/openstack"
 	snapshotcontroller "github.com/kubernetes-incubator/external-storage/snapshot/pkg/controller/snapshot-controller"
 	"github.com/kubernetes-incubator/external-storage/snapshot/pkg/volume"
-	"github.com/kubernetes-incubator/external-storage/snapshot/pkg/volume/aws_ebs"
+	"github.com/kubernetes-incubator/external-storage/snapshot/pkg/volume/awsebs"
```

Comment:
this is annoying

Reply:
Agreed, but what else can we do?  I noticed that persistentvolume and attachdetach are in the same situation.